CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 0.9.7+5
+
+* Change the type on a local function (_onBatch) to reflect the fact that its
+  caller does not statically guarantee its contract.
```

Comment:
Changelogs are generally for downstream users' consumption, and should only include as much information as a consumer of this package will care about. So I'd write something like "Fix an analysis warning." here and save the detailed explanation for the commit message.

Reply:
Done.